src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -546,6 +546,280 @@ def __repr__(self) -> str:
         return repr_str
 
 
+class RandomResizedCrop(tvt_v2.Transform):
+    """Crop the given image to random scale and aspect ratio.
+
+    A crop of random size (default: of 0.08 to 1.0) of the original size and a
+    random aspect ratio (default: of 3/4 to 4/3) of the original aspect ratio
+    is made. This crop is finally resized to given size.
+
+    **Required Keys:**
+
+    - img
+
+    **Modified Keys:**
+
+    - img
+    - img_shape
```

Comment:
We could remove these.

Reply:
Thanks - Done. https://github.com/openvinotoolkit/training_extensions/pull/3352/commits/b8bd40a445bfa5282d192b1555f7a30385382c1c